.circleci/config.yml

```diff
@@ -789,8 +789,7 @@ jobs:
           when: on_success
 
   build-docs-website:
-    docker:
-      - image: grafana/build-container:1.2.13
+    executor: grafana-build
```

Comment:
Why is this step using the build-container? Unrelated to these changes, but maybe something to look into.

Reply:
@marefr Not sure, maybe it was done without good reason.

Reply:
Yeah probably :smile: